testdata/hook.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 # Dummy hook, prints the payload path to a log file.
-echo $1 >> /tmp/hook.log
+echo "$1" >> /tmp/hook.log
```

Comment:
omg we have frôlé the catastrophe

Reply:
Dont tell me about it, it was moins une.